### DISCUSSION TITLE
FormStepsControl doesn't appear to be working

### DISCUSSION THREAD

**Discussion by maxwellsmart84** [vueform/vueform#154] [Questions]
I dont believe FormStepsControl is working as intended.  I have copied the code from the documentation and the text for all the buttons is not rendering:

https://vueform.com/reference/form-steps-control#options

See sandbox:

[Link to sandbox here](https://codesandbox.io/p/devbox/clever-breeze-rltyp7?workspaceId=cdf4d3fd-a871-409c-856b-788734730dd5&embed=1&file=%2Fsrc%2FApp.vue&layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clt34cjy700073b6k4k7p0d6q%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clt34cjy600023b6kovudrxye%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clt34cjy600043b6kohetfiy7%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clt34cjy700063b6kldizizbw%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B52.322511215251495%252C47.677488784748505%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clt34cjy600023b6kovudrxye%2522%253A%257B%2522id%2522%253A%2522clt34cjy600023b6kovudrxye%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clt34egj8004p3b6kfes5fxqk%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A9%252C%2522startColumn%2522%253A9%252C%2522endLineNumber%2522%253A9%252C%2522endColumn%2522%253A9%257D%255D%252C%2522filepath%2522%253A%2522%252Fsrc%252FApp.vue%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clt34egj8004p3b6kfes5fxqk%2522%257D%252C%2522clt34cjy700063b6kldizizbw%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clt34cjy600053b6kftxx2jhu%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A5173%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clt34cjy700063b6kldizizbw%2522%252C%2522activeTabId%2522%253A%2522clt34cjy600053b6kftxx2jhu%2522%257D%252C%2522clt34cjy600043b6kohetfiy7%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clt34cjy600033b6kuwkkzhtl%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%257D%255D%252C%2522id%2522%253A%2522clt34cjy600043b6kohetfiy7%2522%252C%2522activeTabId%2522%253A%2522clt34cjy600033b6kuwkkzhtl%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

**Comment by maxwellsmart84** [comment]
PS - if you use the template slots it appears to work just fine, however I don't believe this is the intended design.

**Comment by adamberecz** [comment]
Actually the docs is wrong and using slots is the right way:

```vue
<FormStepsControls :labels="false">
  <template #previous>
    <<
  </template>
  <template #next>
    >>
  </template>
  <template #finish>
    Finish
  </template>
</FormStepsControls>
```
*(note that if your form has only one step only Previous and Finish buttons will be rendered)*
